Source/dom/models/Override.js

```diff
@@ -14,7 +14,7 @@ Factory.registerClass(Override, MSAvailableOverride)
 
 Override.define('path', {
   get() {
-    return String(this._object.overridePoint().layerIDPath())
+    return String(this._object.overridePoint().path())
```

Comment:
Sorry - I changed that. I thought I'd updated the API stuff, but I guess not...

Reply:
I'm actually not sure how it passed the tests. They were failing for me /shrug